cmd/imagedigestexporter/main.go

```diff
@@ -65,5 +65,18 @@ func main() {
 	if err != nil {
 		log.Fatalf("Unexpected error converting images to json %v: %v", output, err)
 	}
-	fmt.Println(string(imagesJSON))
+	log.Printf("Image digest exporter output: %s ", string(imagesJSON))
+	f, err := os.OpenFile("/builder/termination-log", os.O_WRONLY|os.O_CREATE, 0666)
```

Comment:
could we pass the path to write the termination log to into this binary? (i.e. minimize the number of places we hardcode this path)

Reply:
also would it make sense to put this file into the same dir that the rest of the image related files go in? (/builder/home/image-outputs/)